internal/kafka/command_cluster_update.go

```diff
@@ -27,16 +28,21 @@ func (c *clusterCommand) newUpdateCommand() *cobra.Command {
 				Text: "Update the name and CKU count of a Kafka cluster:",
 				Code: `confluent kafka cluster update lkc-123456 --name "New Cluster Name" --cku 3`,
 			},
+			examples.Example{
+				Text: "Update the type of a Kafka cluster from 'Basic' to 'Standard':",
```

Comment:
Nitpick: we usually put normal quotes around specific values
```suggestion
				Text: `Update the type of a Kafka cluster from "Basic" to "Standard":`,
```